cmd/integration/integration.go

```diff
@@ -186,7 +199,7 @@ func startComponents(manifestURL string) (apiServerURL string) {
 
 	nodeResources := &api.NodeResources{}
 	nodeController := nodeControllerPkg.NewNodeController(nil, "", machineList, nodeResources, cl)
-	nodeController.Run(10 * time.Second)
+	nodeController.Run(5 * time.Second)
```

Comment:
Yeah, node controller now needs to run in integration test to mark nodes as healthy.  Before, we were completely ignoring that whole path (disgustingly so).


> On Jan 29, 2015, at 8:44 PM, Brian Grant <notifications@github.com> wrote:
> 
> In cmd/integration/integration.go:
> 
> > @@ -186,7 +199,7 @@ func startComponents(manifestURL string) (apiServerURL string) {
> >  
> >  	nodeResources := &api.NodeResources{}
> >  	nodeController := nodeControllerPkg.NewNodeController(nil, "", machineList, nodeResources, cl)
> > -	nodeController.Run(10 * time.Second)
> > +	nodeController.Run(5 * time.Second)
> Intended to be in this PR?
> 
> —
> Reply to this email directly or view it on GitHub.
>